static/src/js/components/GranuleResults/GranuleResultsItem.jsx

```diff
@@ -241,6 +250,7 @@ const GranuleResultsItem = forwardRef(({
                       <Button
                         className="button granule-results-item__button granule-results-item__button--add"
                         label="Add granule"
+                        aria-label="Add granule"
```

Comment:
Looking at the `Button` component, it looks like the label prop should be setting the `aria-label`, so you should not need this additional `aria-label`. I'm not sure that its even doing anything at the moment since `Button` does not accept a `aria-label` prop.